PICMI_Python/particles.py

```diff
@@ -282,6 +282,57 @@ def __init__(self, density,
         self.handle_init(kw)
 
 
+class PICMI_UniformFluxDistribution(_ClassWithInit):
+    """
+    Describes a flux of particles emitted from a plane
+
+    Parameters
+    ----------
+    flux: float
+        Flux of particles [m^-2.s^-1]
+
+    flux_normal_axis: string
+        x, y, or z for 3D, x or z for 2D, or r, t, or z in RZ geometry
+
+    surface_flux_position: double
+        location of the injection plane [m] along the direction
+        specified by `flux_normal_axis`
+
+    flux_direction: int
+        Direction of the flux relative to the plane: -1 or +1
+
+    lower_bound: vector of floats, optional
+        Lower bound of the distribution [m]
+
+    upper_bound: vector of floats, optional
+        Upper bound of the distribution [m]
+
+    rms_velocity: vector of floats, default=[0.,0.,0.]
+        Thermal velocity spread [m/s]
+
+    directed_velocity: vector of floats, default=[0.,0.,0.]
+        Directed, average, velocity [m/s]
+    """
+
+    def __init__(self, flux, flux_normal_axis,
+                 surface_flux_position, flux_direction,
+                 lower_bound = [None,None,None],
+                 upper_bound = [None,None,None],
+                 rms_velocity = [0.,0.,0.],
+                 directed_velocity = [0.,0.,0.],
+                 **kw):
+        self.flux = flux
+        self.flux_normal_axis = flux_normal_axis
+        self.surface_flux_position = surface_flux_position
+        self.flux_direction = flux_direction
+        self.lower_bound = lower_bound
+        self.upper_bound = upper_bound
+        self.rms_velocity = rms_velocity
+        self.directed_velocity = directed_velocity
+        self.fill_in = fill_in
```

Comment:
This line is presumably left over from a copy and paste. Otherwise, this looks good.